omnibus/config/software/opscode-pushy-client.rb

```diff
@@ -53,6 +53,13 @@
 dependency "libzmq"
 
 # Core Requirements
+
+# Instead of trying to get all of FFI working on AIX we decided to build a custom C extension for the parts of
+# ZeroMQ that we need. We forked an old C native extension of LibZMQ, https://github.com/chef/rbzmq, and modified
+# it to fit our needs. This meant ripping out any unused code and updating only the methods we need
+# (like context.socket) to support later versions of Ruby and LibZMQ.
+dependency "rbzmq" if aix?
```

Comment:
Could have
```
if aiz?
  dependency "rbzmq"
else
  dependency "libzmq"
end
```

I think this would make it a little more clear that we use different libraries for the different OSes.

Reply:
:doh: ignore this comment. I forgot `ffi-rbzmq` is managed as a gem not an omnibus dependency. I got `libzmq` from line 53 above but thats the underlying gem rbzmq depends on.

Reply:
rbzmq is replacing ffi-rzmq, not libzmq. in fact rbzmq depends on libzmq

https://github.com/chef/omnibus-software/blob/master/config/software/rbzmq.rb#L21

I discussed this with @tyler-ball and he agreed we can leave this as is.